src/components/WalletModal/WalletIcons/TokenPocket.tsx

```diff
@@ -2,139 +2,31 @@ import React from 'react';
 
 const TokenPocket = () => {
     return (
-        <svg
-            xmlns="http://www.w3.org/2000/svg"
-            width="40"
-            height="40"
-            version="1.1"
-            viewBox="0 0 264.583 264.583"
-        >
+        <svg width="40" height="40" viewBox="0 0 1024 1024" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <mask id="mask0_408_225" style="mask-type:alpha" maskUnits="userSpaceOnUse" x="0" y="0" width="1024" height="1024">
```

Comment:
string type in `style` is not allowed. 

Is it needed?

Reply:
Just deleted the `mask` element. I think all of it is not needed.